src/constants.ts

```diff
@@ -507,9 +507,10 @@ function getFwVersionConst (v: Buffer): FirmwareConstants {
     c.eip712MaxTypeParams = 36;
   }
   // V0.14.1 Added the Terra decoder
```

Comment:
Should we just remove this entirely?

Reply:
No I'd like to keep it as reference to document the enum values. Also we will likely re-use a lot of it if/when we do Cosmos